File: graves/codeforces/2A/YoungPhysicist.py
import sys


############ ---- Input Functions ---- ############
def single_integer_input():
    return int(sys.stdin.readline())


def array_of_integers_input():
    return list(map(int, sys.stdin.readline().split()))


def array_of_chars_input():
    s = sys.stdin.readline()
    return list(s[: len(s) - 1])


def space_separated_integers_input():
    return map(int, sys.stdin.readline().split())


if __name__ == "__main__":
    n = single_integer_input()
    
    sum_x = 0
    sum_y = 0
    sum_z = 0

    for i in range(n):
        vector = list(space_separated_integers_input())
        
        sum_x += vector[0]
        sum_y += vector[1]
        sum_z += vector[2]

    if sum_x == 0 and sum_y == 0 and sum_z == 0:
        print("YES")
    else:
        print("NO")
        
        

File: graves/codeforces/2A/YoungPhysicist.js
'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', (inputStdin) => {
  inputString += inputStdin;
});

process.stdin.on('end', (_) => {
  inputString = inputString
    .trim()
    .split('\n')
    .map((string) => {
      return string.trim();
    });

  main();
});

function readline() {
  return inputString[currentLine++];
}

function main() {
  var T = readline();

  let matrix = [];

  while (T-- > 0) {
    let vector = readline()
      .split(' ')
      .map((int) => parseInt(int));

    matrix.push(vector);
  }

  let sumX = 0;
  let sumY = 0;
  let sumZ = 0;

  for (let i = 0; i < matrix.length; i++) {
    let vector = matrix[i];
    sumX += vector[0];
    sumY += vector[1];
    sumZ += vector[2];
  }

  if (sumX === 0 && sumY === 0 && sumZ === 0) {
    console.log('YES');
  } else {
    console.log('NO');
  }
}
